File: PerfTester-Xamarin/PerfTester-Xamarin/Screens/TestList/TestListViewModel.cs
using System;
using PerfTesterXamarin.Models;

namespace PerfTesterXamarin.Screens.TestList
{
    public class TestListViewModel
    {
        public Test[] AllTests = new Test[] 
        { 
            new SimpleLoopTest(),
            new RandomizingTest(),
            new SimpleCalculationsSyncTest(),
            new SimpleCalculationsAsyncTest(),
            new HeavySyncCalculationsTest(),
            new HeavyAsyncCalculationsTest(),
            new SimpleDatabaseInsertingTest(),
            new ComplexDatabaseInsertingTest(),
            new SimpleDatabaseLoadingTest(),
            new AssetLoadingTest(),
            new LoadingControllerTest(),
            new LoadingComplicatedControllerTest(),
            new UserDefaultsSaveTest(),
            new UserDefaultsLoadTest(),
            new JsonDeserializationTest(),
            new MultithreadingTest()
        };
    }
}

File: PerfTester-Xcode/PerfTester-Xcode/Screens/TestList/TestListViewModel.swift
//
//  TestListViewModel.swift
//  PerfTester-Xcode
//
//  Created by Patryk Średziński on 08.06.2018.
//  Copyright © 2018 private.perftesterxcode. All rights reserved.
//

import UIKit

class TestListViewModel {
    let allTests = [
        SimpleLoopTest(),
        RandomizingTest(),
        SimpleCalculationsSyncTest(),
        SimpleCalculationsAsyncTest(),
        HeavyCalculationsSyncTest(),
        HeavyCalculationsAsyncTest(),
        SimpleDatabaseInsertingTest(),
        ComplexDatabaseInsertingTest(),
        SimpleDatabaseLoadingTest(),
        AssetLoadingTest(),
        LoadingControllerTest(),
        LoadingComplicatedControllerTest(),
        UserDefaultsSaveTest(),
        UserDefaultsLoadTest(),
        JsonDeserializationTest(),
        MultithreadingTest()
    ]
}
